Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift

```diff
@@ -82,11 +82,10 @@ struct BasicAppSettingsView: View {
                 }
             }
         } else {
-            NavigationLink(destination: EnterSeedPhraseView(rootIsActive: self.$addOwnerIsActive),
+            NavigationLink(destination: EnterSeedPhraseView(),
                            isActive: self.$addOwnerIsActive) {
```

Comment:
do we still need addOwnerIsActive?

Reply:
Just tried it, and yes we don't need it anymore